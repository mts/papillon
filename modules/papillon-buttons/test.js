import React from 'react';
import {
  btn
} from './flavor';

describe('<Button />', () => {
  let wrapper;

  describe('Instance', () => {
    test('must be an instance of Button', () => {
      wrapper = renderMount(btn);
      expect(wrapper.find('Button')).toHaveLength(1);
    });
  });

  describe('Snaphot', () => {
    test('must match btn', () => {
      expect(global.renderToJSON(btn)).toMatchSnapshot();
    });
  });
});
