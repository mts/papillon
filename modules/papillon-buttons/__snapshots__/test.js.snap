// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Snaphot must match btn 1`] = `
<div>
  <button
    className="btn"
    type="button"
  >
    Button
  </button>
</div>
`;
